### ISSUE TITLE
useless-super-delegation with ABC

### ISSUE THREAD

**Issue by rickwierenga** [PyCQA/pylint#7435]
### Bug description

Subclassing an ABC and just calling parent raises `useless-super-delegation`.

Example:

```py
from abc import ABC, abstractmethod

class MyABC(ABC):
  @abstractmethod
  def do_something(self):
    print("Some implementation!")

class MyConcrete(MyABC):
  def do_something(self):
    super().do_something()
```

### Command used

```shell
pylint a.py
```

### Pylint output

```shell
************* Module a
a.py:1:0: C0114: Missing module docstring (missing-module-docstring)
a.py:9:2: W0235: Useless super delegation in method 'do_something' (useless-super-delegation)

-----------------------------------
Your code has been rated at 7.14/10
```

### Expected behavior

I guess you'd not want to raise the `useless-super-delegation` since this delegation is not useless.

### Pylint version

```shell
pylint 2.14.3
astroid 2.11.6
Python 3.9.13 (main, May 24 2022, 21:13:51) 
[Clang 13.1.6 (clang-1316.0.21.2)]
```

### OS / Environment

macOS 12.3.1

**Comment by Pierre-Sassoulas** [comment]
Why are you decorating the function as abstract in this example if you're using it later ? I think an abstract function should not have any behavior implemented, this is probably why we have a message that feel wrong here.

**Comment by mbyrnepr2** [comment]
I see how this could be a false positive. From [PEP3119](https://peps.python.org/pep-3119/):

_Unlike Java’s abstract methods or C++’s pure abstract methods, abstract methods as defined here may have an implementation. This implementation can be called via the super mechanism from the class that overrides it. This could be useful as an end-point for a super-call in framework using cooperative multiple-inheritance_
